.ahoy.yml

```
@@ -36,9 +36,47 @@ commands:
+      clear
+      docker compose down
+      npm run build
+      docker compose up -d
+      clear
+
+  logs:
+    usage: Show the docker logs
+    cmd: |
+      clear
+      docker logs sentinel-test
+      docker logs cluster-test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)